decidim-core/decidim-core.gemspec

```diff
@@ -40,7 +40,7 @@ Gem::Specification.new do |s|
   s.add_dependency "paper_trail", "~> 8.0"
   s.add_dependency "pg", ">= 0.18", "< 2"
   s.add_dependency "premailer-rails", "~> 1.9"
-  s.add_dependency "rails", "~> 5.1", "< 5.2"
+  s.add_dependency "rails", "~> 5.1.5"
```

Comment:
I would leave it as it is, why put so much pressure on end applications to upgrade? It seems too much for me with no real benefit.

Reply:
Not exactly - `rails 5.1.5` [fixes the `pg-1.0.0` integration](https://github.com/rails/rails/issues/31673) so if we want to keep `pg` compatible, we need to upgrade it!

Reply:
No, we already support pg 1.0.0 and Rails 5.1.5, that was my previous PR!

Reply:
Oh! But I mean - `rails-5.1.4` doesn't work with `pg-1.0.0`, am I wrong? This could lead to errors on existing apps if they (for some reason) update `pg` to `1.0.0` and not rails.

(BTW, I hadn't seen your previous PR, this is getting out of hand 🤣 )

Reply:
No, it doesn't. But the situation you mention won't happen, bundler will just do the right thing

Reply:
I mean, if you specify conflicting dependencies in your Gemfile, it will raise an error and let you know why. If you `bundle update`, it will update both if the specifications in the `Gemfile` allow it.

Reply:
Oh, really? I'm lost here. Rails `5.1.4` wasn't locked to `pg <1.0.0` so it was technically possible to upgrade `pg` and having `rails` break. Imagine you *have* locked `rails 5.1.4` and `bundle update` - you could have that issue.

I agree on that this is an unlikely scenario, but looks kinda broken by design, right?

Reply:
Ah, yes, that edge case could happen, you're right! Only 100% untested apps would leak it but it's a bug in Rails version specifications that we can choose to prevent :+1:.